Makefile

```diff
@@ -73,5 +73,6 @@ endif
 
 simulator:
 	CGO_ENABLED=0 go build -o bin/simulator cmd/simulator/main.go
+	bin/simulator
```

Comment:
em I think mostly, we do build in Makefile, not include running it.

Reply:
Any suggestion?